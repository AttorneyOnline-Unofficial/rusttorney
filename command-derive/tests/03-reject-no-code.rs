use command_derive::*;

#[derive(Command)]
enum ClientRequest {
    #[command(code = "HI", handle = "handle_handshake")]
    Handshake {
        hdid: String
    },
    Ping
}

fn main() {}
