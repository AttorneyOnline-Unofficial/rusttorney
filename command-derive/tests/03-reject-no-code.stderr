error: No `code` parameter on ClientRequest::Ping
 --> $DIR/03-reject-no-code.rs:3:10
  |
3 | #[derive(Command)]
  |          ^^^^^^^
  |
  = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)
